app/main.py

```diff
@@ -1 +1,38 @@
-#  write your code here
+class Animal():
+    def __init__(self, name: str,
+                 appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> str:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        return 0
```

Comment:
The `feed` method of the `Animal` class has an unnecessary `else` statement. According to the checklist, we should avoid using an `else` statement if it's not needed. In this case, you can simply write `return 0` after the `if` statement block.